### ISSUE TITLE
Translucent tiles change colors on mouse move when picking

### ISSUE THREAD

**Issue by ebogo1** [CesiumGS/cesium#9420]
Sandcastle example: [link](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRtb9s4DP4rQr7MxmWKX+O4zYrl0gANsKxFk+1wOA+FaiuNUFkyJNldNuy/H2XHadLdfbEt8SH5kHzohijUMPpCFfqABH1Bc6pZXeKv7Z3zLm+PcykMYYKqd0P0MxMIGaoUXNwp2bCCqoveLVeUGPqXVLzYdBDHzcQv9zITmWggl2GcamrOk3Wv8HrTGZ2fqFb8GHMpxT3VslY5xVsly5kGzLJwxl4SuaiN3VWAdU4FxZViJTOsoRqTonAOGTsKhwPWZs/p/5NYW7PTVppLLhW6QNlAPT0SJ4jjIXp9eDh2s8Gxwj48dKHYQ3NKpik2OyqcbS1ywyS0o41qW/Eoa1Ew8bSudlRZMr33ueGyxXcF5qSkimB7+vMM5Jz7DE8LuwEyYLljJt/dE/FEHQ97Q/QeuA/fkMAKoLVGf6DYA5Dr/kdyLuXzzGwUEXorVekcsqyIUex7hJfXi8+b5eZv63oy90ry/ZMUUOUhGoVmGXaYUduTHnLRSQyhHYN8Kt/t4eqknrsOd9NbewZzogx8ERG2MrmHUoDjDFS4d/557+PQT6NwnAZJEvqRFwR2euN0MvHGQRx6qWff0JYWGKZxnE68wIsCPzkCUz9OJkniB3H0Cgz9IPDSOA3HPTCBNGEQpd5kfAwYhfEkCCIviiNvcsQlfjqO43icppNvrjvsyi5hgxQjHB03YG41iO8X1/iFmd2MVzviWOUdHHJOtGZblhOrsM2+oieuv9nwfLFeflk9hNcPm+WnhQ3x63VU/SIpSsW6IjldNDCpGyIKbu9h8URVm1mn5V7USIqVrDVdyYY6NmAJHyX4ZcLthnm+oCx/dnoICKG4k5rZOK1moKgD9/UbFi371e2X9eJhdft14V4OhoNpu8lXXSM+srKSyti/h4PxyNCy4tBMPXqs82dYrFxrm2I66p2mBWsQKz5kgzf/uWzQdRUs25rzNftBs8HVdAT4Mzcu2926bajiZG8hO//qU3eJMZ6O4Pi7l5GSPxJ1EvFf)

Browser: Firefox

Operating System: Linux

Found by @lilleyse in #9399. From a comment in that PR: 
![e](https://user-images.githubusercontent.com/31491650/110488047-23eeed80-80bc-11eb-8930-293da673285c.gif)

**Comment by ebogo1** [comment]
Some notes - 

@lilleyse Suggested that this was due to a frame buffer not being cleared, and after testing I'm pretty confident this is true. When translucency in the above Sandcastle is set to `0.01` instead of `0.5`, the ghosting effect of old frame buffers loading back in is much more noticeable and gradual.

This also seems to happen when an old frame buffer is present and the camera moves, and results in a ghosting artifact like this - 
![image](https://user-images.githubusercontent.com/31491650/111844633-c6d41280-88d9-11eb-8331-01bd80f38cd0.png)

**Comment by ebogo1** [comment]
I found something that might be of note - 
1. `Scene.executeCommands` calls `TranslucentTileClassification.executeTranslucentCommands` and `TranslucentTileClassification.executeClassificationCommands`, in that order.
2. The translucent commands function increments `TranslucentTileClassification._frustumsDrawn` by +=1 if `hasTranslucentDepth` is true. All the other places where `frustumsDrawn` is changed set it to 0.
3. In `TranslucentTileClassification.executeClassificationCommands`, the `drawClassificationFBO` only gets cleared if `frustumsDrawn` is greater than 1. Setting the check to `>=` instead of `>` appears to fix this bug, and does not seem to impact correctness of `Scene.pick`'s return value in the above Sandcastle. 

Is it possible for `frustumsDrawn` to become greater than 1? In `Scene.executeCommandsInViewport`, it is set to 0 in `updateAndClearFramebuffers` before the call to `executeCommands`, so at first glance it looks like it it isn't. @lilleyse @likangning93 Any thoughts, maybe I'm missing something?

**Comment by lilleyse** [comment]
I don't remember all the details but I believe `frustumsDrawn` is there in case the scene renders multiple frustums like when log depth is disabled (`scene.logarithmicDepthBuffer = false`)

**Comment by ebogo1** [comment]
Update - I believe this bug happens because `Picking.pick` might call `TranslucentTileClassification`'s execute commands before the classification's `clear` function has a chance to run during the last render.

- `TranslucentTileClassification` stores an `accumulationFBO` if there are more than one frustums using the classification. In the `clear` function, this is handled by always clearing the `drawClassificationFBO` and only clearing the `accumulationFBO` if `frustumsDrawn > 1`.
- The check `if (this._frustumsDrawn > 1)` in `executeClassificationCommands` before clearing the looks like an optimization to avoid clearing the classification FBO both in the execute function and in the clear function. 
- It is possible for the picking function to call the two execute functions in `TranslucentTileClassifications` in between the time that the render executes the translucent classification commands and clears the buffer for the next frame. This is because `Scene` calls `updateAndClearFrameBuffers` before it calls `executeCommands` in `executeCommandsInViewport`. 

I wasn't sure what the best way to test this was but I figured that with the above thinking, the bug could be replicated outside of picking by only clearing the classification FBO in `clear` if `frustumsDrawn > 1`. This would simulate the behavior of not clearing the FBO in between consecutive calls to `executeClassificationCommands`. With this change, the same ghosting artifacts can be seen because the uncleared FBO stacks on itself without picking - 
![image](https://user-images.githubusercontent.com/31491650/112066261-6690db00-8b3c-11eb-992b-6191a02dbb4c.png)

A better solution might be to force the `pick` function to clear the classification FBO if the commands are being executed as part of a pick rather than a render. I'm not sure how easy of a change that'd be, so for now I opened a PR with the simpler change - https://github.com/CesiumGS/cesium/pull/9440.
